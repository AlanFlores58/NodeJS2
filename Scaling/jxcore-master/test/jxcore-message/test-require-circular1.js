// Copyright & License details are available under JXCORE_LICENSE file

console.log("index1.js");
var m = require("./assets/_require-circular2.js");
console.log("obj", m.obj);