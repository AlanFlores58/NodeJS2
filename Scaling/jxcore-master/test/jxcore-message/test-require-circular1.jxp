{
	"name": "test-require-circular1",
	"version": "1.0",
	"author": "",
	"description": "",
	"company": "",
	"website": "",
	"package": null,
	"startup": "test-require-circular1.js",
	"execute": null,
	"extract": false,
	"output": "test-require-circular1.jx",
	"files": [
		"test-require-circular1.js",
		"assets/_require-circular2.js"
	],
	"assets": [],
	"preInstall": null,
	"library": true,
	"fs_reach_sources": true
}