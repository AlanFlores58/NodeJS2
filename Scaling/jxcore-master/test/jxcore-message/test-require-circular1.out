index1.js
index2.js
obj {}